core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -322,28 +323,33 @@ def detect_authentication_to_install
     def detect_payment_method_to_install
       return 'paypal' if Bundler.locked_gems.dependencies['solidus_paypal_commerce_platform']
       return 'bolt' if Bundler.locked_gems.dependencies['solidus_bolt']
+      return 'braintree' if Bundler.locked_gems.dependencies['solidus_braintree']
 
-      descriptions = {
-        paypal: "- [#{set_color 'paypal', :bold}] Install `solidus_paypal_commerce_platform` (#{set_color :default, :bold}).",
-        bolt: "- [#{set_color 'bolt', :bold}] Install `solidus_bolt`.",
-        none: "- [#{set_color 'none', :bold}] Skip installing a payment method.",
+      allowable_payment_methods_hash = {
+        'none' => ['paypal', 'braintree', 'none'],
+        'classic' => ['paypal', 'bolt', 'none'],
```

Comment:
SolidusBraintree 1.x is currently only compatible with Solidus 2.x, so if we want Solidus 3.4 to install SolidusBraintree with classic frontend, we'll need to update SolidusBraintree 1.x to be compatible with Solidus 3.4. 

@kennyadsl Is that something we want to do?